src/org/jitsi/impl/neomedia/VideoMediaStreamImpl.java

```diff
@@ -448,8 +448,8 @@ else if(closestAspect == null)
      * The <tt>RemoteBitrateEstimator</tt> which computes bitrate estimates for
      * the incoming RTP streams.
      */
-    private final RemoteBitrateEstimator remoteBitrateEstimator
-        = new RemoteBitrateEstimatorSingleStream(
+    private final RemoteBitrateEstimatorWrapper remoteBitrateEstimator
+        = new RemoteBitrateEstimatorWrapper(
```

Comment:
i'm confused that we have a different type here (`RemoteBitrateEstimatorWrapper` instead of `RemoteBitrateEstimator`) given that we initialize the 'wrapper' the same way as we did the estimator itself, is the 'wrapper' effectively an estimator?